doc/command_ref.rst

```diff
@@ -136,13 +136,16 @@ Options
 
     DNF uses a separate cache for each user under which it executes. The cache for the root user is called the system cache. This switch allows a regular user read-only access to the system cache which usually is more fresh than the user's and thus he does not have to wait for metadata sync.
 
+``--color=<color>``
+    control whether color is used
```

Comment:
Do we have any working default color settings? Because it'd be nice if this actually did something...

Reply:
I've got this and a few other argument option documentation fixes on a local branch... There's this bug for it which I've been fixing: https://bugzilla.redhat.com/show_bug.cgi?id=1673278

It does seem I've missed a couple of options you didn't, perhaps we can go with your patch. If we do, can you use this description for the --color option:

```
Control whether color is used in terminal output. Valid values are ``always``, ``never`` and ``auto`` (default).
```

Reply:
Thanks for the --color description, I added it there.

Reply:
@Conan-Kudo, I know of only one use of color: ``dnf repolist --all`` will color differently "disabled" and "enabled". Not sure where it's used beside that.

Reply:
You can set the color for the package manager.

Try with this:
```
color_list_installed_older=yellow
color_list_installed_newer=bold,yellow
color_list_installed_reinstall=dim,cyan
color_list_installed_running_kernel=bold,underline
color_list_installed_extra=bold,red
color_list_available_upgrade=bold,blue
color_list_available_downgrade=dim,magenta
color_list_available_install=bold,cyan
color_list_available_reinstall=bold,underline,green
color_list_available_running_kernel=bold,underline
color_search_match=bold,magenta
color_update_installed=dim,red
color_update_local=dim,green
color_update_remote=bold,green
```

Reply:
You can set the color for the package manager.

Try with this added to `/etc/dnf/dnf.conf`:
```
color_list_installed_older=yellow
color_list_installed_newer=bold,yellow
color_list_installed_reinstall=dim,cyan
color_list_installed_running_kernel=bold,underline
color_list_installed_extra=bold,red
color_list_available_upgrade=bold,blue
color_list_available_downgrade=dim,magenta
color_list_available_install=bold,cyan
color_list_available_reinstall=bold,underline,green
color_list_available_running_kernel=bold,underline
color_search_match=bold,magenta
color_update_installed=dim,red
color_update_local=dim,green
color_update_remote=bold,green
```

Reply:
@Conan-Kudo hey, nice, suddenly it looks much better :smile: So we have no colors by default except for that one odd command @pkratoch mentioned? That's a bit sad...

Reply:
We should actually set these colors by default... I think there was a plan a few years ago to do it, and then it got forgotten.